src/thermo/Elements.cpp

```diff
@@ -252,25 +327,23 @@ double getElementWeight(int atomicNumber)
 
 string getElementSymbol(const std::string& ename)
 {
-    int numElements = numElementsDefined();
-    int numIsotopes = numIsotopesDefined();
     string name = toLowerCopy(trimCopy(ename));
-    for (int i = 0; i < numElements; i++) {
-        if (name == atomicWeightTable[i].fullName) {
-            return atomicWeightTable[i].symbol;
+    for (const auto& atom : atomicWeightTable) {
+        if (name == atom.fullName) {
+            return atom.symbol;
         }
```

Comment:
If the suggested `elementNames` map is adopted, it could be used here (same for a potential `isotopeNames` map below).

Reply:
I can see how this would be simpler, but I think the complexity would just move the the creation of the map, I don't think it would actually remove complexity overall...